pkg/builders/catalogsourcebuilder.go

```diff
@@ -62,10 +62,14 @@ func (b *CatalogSourceBuilder) WithOLMLabels(cscLabels map[string]string) *Catal
 }
 
 // WithOwnerLabel sets the owner label of the CatalogSource object to the given owner.
-func (b *CatalogSourceBuilder) WithOwnerLabel(name, namespace string) *CatalogSourceBuilder {
-	labels := map[string]string{
-		OwnerNameLabel:      name,
-		OwnerNamespaceLabel: namespace,
+func (b *CatalogSourceBuilder) WithOwnerLabel(name, namespace string, isOpsrcOwner bool) *CatalogSourceBuilder {
```

Comment:
Using the boolean `isOpsrcOwner` here seems like an antipattern to me. One of the value adds to the builder pattern is that you can make these methods cumulative. I think adding the `OpsrcOwner` or `Owner` label should just be separated into separate methods so it is clearer and more explicit as to exactly what is being set.

Reply:
Also, apply this comment everywhere that you are using this pattern in the PR.

Reply:
Yeah that makes sense